packages/mdc-slider/foundation.js

```diff
@@ -357,7 +357,7 @@ class MDCSliderFoundation extends MDCFoundation {
 
   /**
    * Returns the computed name of the event
-   * @param {Event} kbdEvt
+   * @param {?Event} kbdEvt
```

Comment:
The function body appears to assume this param is non-null, so we should mark it as such in the JSDoc.

Reply:
So make `kbdEvt` non-nullable, is that correct?